vendor/k8s.io/kube-openapi/pkg/util/trie.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Yes, it is duplicate. It is a simple utility that used in both apiserver and here. I cannot import it from API server as it creates a dependency cycle. The only other option is to create a util repo and move it there, that would be overkill for this simple utility but when we had that repo for other utilities, we can move this there and remove duplicates.